packages/framework/src/Model/Administrator/Security/AdministratorRolesChangedFacade.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Model\Administrator\Security;
+
+use Shopsys\FrameworkBundle\Model\Administrator\Administrator;
+use Shopsys\FrameworkBundle\Model\Administrator\AdministratorFacade;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
+
+class AdministratorRolesChangedFacade
+{
+    /**
+     * @var \Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface
+     */
+    protected $tokenStorage;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Administrator\AdministratorFacade
+     */
+    protected $administratorFacade;
+
+    /**
+     * @param \Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface $tokenStorage
+     * @param \Shopsys\FrameworkBundle\Model\Administrator\AdministratorFacade $administratorFacade
+     */
+    public function __construct(TokenStorageInterface $tokenStorage, AdministratorFacade $administratorFacade)
+    {
+        $this->tokenStorage = $tokenStorage;
+        $this->administratorFacade = $administratorFacade;
+    }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Administrator\Administrator $administrator
+     */
+    public function refreshAdministratorToke(Administrator $administrator): void
```

Comment:
```diff
-  public function refreshAdministratorToke(Administrator $administrator): void
+  public function refreshAdministratorToken(Administrator $administrator): void
```

not writing the suggestion as usages have to be changed too

Reply:
fixed